scripts/windows/efcore/add_migration.ps1

```diff
@@ -19,7 +18,7 @@ function AddMigration {
 
     $migrationProject = "$repoRoot\Modules\$moduleName\src\$moduleName.Infrastructure.Database.$provider"
 
-    $cmd = "dotnet ef migrations add --startup-project '$startupProject' --project '$migrationProject' --context $dbContextName --output-dir Migrations --verbose $migrationName"
+    $cmd = "dotnet ef migrations add --startup-project '$startupProject' --project '$migrationProject' --context $dbContextName --output-dir Migrations --verbose $migrationName -- --environment $environment"
```

Comment:
Why don't you set ASPNETCORE_ENVIRONMENT anymore? I like your new approach more - but only if it actually works. :) I guess you tried it out?

Reply:
I tried it out for all scripts and both databases, everything works properly.
Regarding ASPNETCORE_ENVIRONMENT, I found the success rate a bit wonky, which was really odd.
I was even able to find multiple threads with people reporting the same results (sometimes the dotnet ef migrations command would default to the appsetting.json file).